src/views/password/ForgotPassword.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Grid, Paper, Box, Typography, TextField } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import {
+	sendResetLink,
+	forgotPassword,
+	forgotPasswordError,
+	deleteError,
+} from '../../actions/resetPasswordAction';
+import { useStyles } from '../../styles/Styles';
+
+export class ForgotPassword extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			email: '',
+			error: '',
+		};
+	}
+	handleSubmit(event) {
+		const { deleteError } = this.props;
+		deleteError();
+		event.preventDefault();
+		const email = this.state.email;
+		const { sendResetLink } = this.props;
+
+		sendResetLink(email);
+		this.setState({ error: 'this email is not registered' });
```

Comment:
Can't you get this error from the API response instead of hard coding it? what do you think?

Reply:
@Jajabenit250 Yes I can get the response from API but it is not descriptive enough for the user, that why I am using this one.